chainer_chemistry/dataset/splitters/time_splitter.py

```diff
@@ -0,0 +1,190 @@
+from logging import getLogger
+import os
+import shutil
+
+from chainer.dataset import download
+import numpy
+import pandas
+
+from chainer_chemistry.dataset.splitters.base_splitter import BaseSplitter
+
+
+def get_year_table_filepath(download_if_no_exist=True):
```

Comment:
Is year table related to PDBBind dataset? I want to consider it is possible to make TimeSplitter, independent of specific dataset.